### DISCUSSION TITLE
RSS feed for the official Gatsby blog

### DISCUSSION THREAD

**Discussion by crock** [gatsbyjs/gatsby#27634] [Ideas / Feature Requests]
I can make the PR to add this plugin if you all would like.
https://www.gatsbyjs.com/plugins/gatsby-plugin-feed

**Comment by ascorbic** [comment]
Good idea. We'll look into this

**Comment by kawpii** [comment]
@KyleAMathews @ChristopherBiscardi  

Still there is no RSS feed for Gatsby blog, why is that?
